### PR TITLE
Added more testcases for processTx and disputeBatch function

### LINKED ISSUE #7: Add dispute batch testcases

### PR DESCRIPTION
**Pull request by thrilok209**
Fixes #7

### CODE REVIEW

File: contracts/rollup.sol [hunk 1]
```diff
@@ -174,8 +174,9 @@ contract RollupHelpers is RollupSetup {
 
             // TODO use safe math
             // calculate challeger's reward
-            challengerRewards += (batch.stakeCommitted * 2) / 3;
-            burnedAmount += batch.stakeCommitted.sub(challengerRewards);
+            uint _challengerReward = (batch.stakeCommitted * 2) / 3;
```
Comment (vaibhavchellani) [inline contracts/rollup.sol]: Can we try and remove the TODO here ;)

Comment (thrilok209) [reply contracts/rollup.sol]: Fixed it.

File: test/Rollup.spec.ts [hunk 2]
```diff
@@ -319,26 +414,993 @@ contract("Rollup", async function (accounts) {
...
+    let batchId = await rollupCoreInstance.numOfBatchesSubmitted();
+    let batchMarker = await rollupCoreInstance.invalidBatchMarker();
+    assert.equal(batchMarker, "0", "batchMarker is not zero");
+    assert.equal(batchId -1 , falseBatchTwo.batchId - 1, "batchId doesnt match");
+    Alice.Amount += falseBatchTwo.txs[0].amount;
+    Bob.Amount -= falseBatchTwo.txs[0].amount;
+    Alice.nonce--;
+  })
+
+  // it("submit new batch 4nd", async function () {
```
Comment (vaibhavchellani) [inline test/Rollup.spec.ts]: @thrilok209 can you please remove the commented part if it's not needed, the test file is very long as is 😰

**Comment by vaibhavchellani** [comment]
Great work @thrilok209! Any idea why the CI might be failing?

**Pull request merged**